clearwater-config-manager.root/etc/cron.hourly/clearwater-config-manager-log-cleanup

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+ONE_HUNDRED_MEG=107374182
```

Comment:
Good spot, make sure this variable gets renamed to ONE_HUNDRED_AND_TWO_POINT_FOUR_MEG or whatever is appropriate